relay/paseo/src/lib.rs

```diff
@@ -1481,6 +1523,122 @@ impl pallet_delegated_staking::Config for Runtime {
 	type CoreStaking = Staking;
 }
 
+impl pallet_staking_async_ah_client::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type CurrencyBalance = Balance;
+	type AssetHubOrigin =
+		frame_support::traits::EitherOfDiverse<EnsureRoot<AccountId>, EnsureAssetHub>;
+	type AdminOrigin = EnsureRoot<AccountId>;
+	type SessionInterface = Self;
+	type SendToAssetHub = StakingXcmToAssetHub;
+	#[cfg(feature = "paseo")]
+	type MinimumValidatorSetSize = ConstU32<50>;
+	#[cfg(not(feature = "paseo"))]
+	type MinimumValidatorSetSize = ConstU32<400>;
```

Comment:
This should be 100 in all cases. Paseo has 150 validators, 100 sounds like a good minimum.